lib/ansible/modules/cloud/digital_ocean/digital_ocean_size_facts.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2018, Ansible Project
+# Copyright: (c) 2018, Abhijeet Kasurde <akasurde@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+
+DOCUMENTATION = '''
+---
+module: digital_ocean_size_facts
+short_description: Gather facts about DigitalOcean Sizes
+description:
+    - This module can be used to gather facts about sizes.
```

Comment:
Could we included the word `Droplet Sizes`? That will clear up any confusion on what sizes.